quisp/modules/Application/Application.cc

```diff
@@ -69,7 +69,7 @@ void Application::initialize() {
     EV_INFO << "My connection setup request will be sent from " << my_address << " to " << endnode_dest_addr << "\n";
     ConnectionSetupRequest *pk = createConnectionSetupRequest(endnode_dest_addr, number_of_resources);
     // delay to avoid conflict
-    scheduleAt(simTime() + exponential(0.00001 * my_address), pk);
+    scheduleAt(simTime() + 0.1 * my_address, pk);
```

Comment:
Why change random time to fixed time?

Reply:
Because it was easier to see the simulation. Here should be more generalized with zomi's connection generation though.

Reply:
that's fair